src/js/game/GameController.js

```diff
@@ -277,18 +278,30 @@ class GameController {
       [Phaser.Keyboard.SPACEBAR]: -2
     };
 
+    const shouldIgnoreKeyPress = function () {
+      // Ignore keyboard input when project is being renamed
+      return $('.project_save').length > 0;
```

Comment:
I don't love the idea of polluting the craft repo with specific knowledge about the containing system. What if we did something like

```js
function editableElementSelected () {
  return !(document.activeElement === document.body || document.activeElement === null);
}
```